terraform/20-app/cloud-front.front-end.tf

```diff
@@ -252,3 +252,48 @@ module "cloudfront_password_protection_frontend" {
   create = local.add_password_protection
   name   = "${local.prefix}-front-end-password-protection"
 }
+
+
+################################################################################
+# Response policies
+################################################################################
+resource "aws_cloudfront_response_headers_policy" "front_end" {
```

Comment:
This custom policy mimics the managed `CORS-with-preflight-and-SecurityHeadersPolicy` that we were using previously with the exception of setting `X-Frame-Options` to DENY

Reference: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-response-headers-policies.html#managed-response-headers-policies-cors-preflight-security